bin/unit-tests-coverage.txt

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+#
+# Run the unit tests with code coverage.
+#
+# This ensures that the logic in the VIP sniffs is covered by unit tests.
+#
+# EXAMPLE TO RUN LOCALLY:
+#
+#   ./bin/unit-test-coverage
+#
+# If you want to view the results locally, run with the following extra options:
+#
+#   ./bin/unit-test-coverage --coverage-html ./build/logs/
+#
+
+"$(pwd)/vendor/bin/phpunit" --filter WordPressVIPMinimum "$(pwd)/vendor/squizlabs/php_codesniffer/tests/AllTests.php" %*
```

Comment:
What does the `%*` do here?

Reply:
It will take any additional arguments you pass on the command line and append them.

Without it, the second example of how to run the command (line 13) wouldn't (shouldn't) work as `--coverage-html ./build/logs/` would be thrown away instead of added to the command.

Hmm.. trying to remember now whether this is a Windows syntax or cross-platform. Would you mind testing both the commands ? IIRC you are on Mac or not ?